pkg/netutil/private_subnet.go

```diff
@@ -0,0 +1,51 @@
+package netutil
+
+import "net"
+
+var (
+	// IPV4PrivateNetworks holds RFC1918 addresses plus other reserved IPv4 ranges.
+	IPV4PrivateNetworks = []*net.IPNet{
+		// private networks per RFC1918
+		MustParseCIDR("10.0.0.0/8"),
+		MustParseCIDR("172.16.0.0/12"),
+		MustParseCIDR("192.168.0.0/16"),
+
+		// Other reserved ranges
+		MustParseCIDR("100.64.0.0/10"),
```

Comment:
Just a heads-up that the definition in `central/compliance/checks/common/insecure_registries.go` was missing those. I believe adding them causes no harm, but just wanted to point that out.